opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationRequestCommon.java

```diff
@@ -168,8 +169,8 @@ public SELF user_anton_brueckner_provided_initial_parameters_to_list_accounts_wi
     public SELF user_denied_consent() {
         ExtractableResponse<Response> response = RestAssured
                 .given()
+                    .header(X_REQUEST_ID, DEFAULT_X_REQUEST_ID)
```

Comment:
Consent API calls should use generated request IDs (like it was before)

Reply:
random UUID is generated